src/sql/parts/profiler/editor/profilerInput.ts

```diff
@@ -156,12 +155,12 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 			columnNameMap['cpu_time'] = 'CPU';
 			columnNameMap['duration'] = 'Duration';
 			columnNameMap['logical_reads'] = 'Reads';
+			columnNameMap['event_sequence'] = 'EventSequence';
+			columnNameMap['client_pid'] = 'ClientProcessID';
+			columnNameMap['writes'] = 'Writes';
+			columnNameMap['duration'] = 'Duration';
 
-			for (let idx = 0; idx < columns.length; ++idx) {
-				let columnName = columns[idx];
-				data[columnName] = '';
-			}
-
+			data['TextData'] = ' ';
```

Comment:
Why `' '`? Would `''` (no space) or `undefined` be better?

From the code alone it's not really clear why we need this so I'd suggest adding a comment in any case

Reply:
So it turns out that having undefined crashes the whole thing, and adding a space solves the issue with highlighting text as you scroll through. It looks like the issue was purely when the text field is completely empty it highlights the next displayed text, but a single space fixes it. It's definitely a hacky work around, but I figured it improves the profiler experience and is probably good enough for the June release. I'll be changing a lot of display logic for the July release, so I'm planning to tackle this more permanently then.

Reply:
Sounds good